src/cascadia/TerminalApp/Jumplist.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// Module Name:
+// - Jumplist.h
+//
+// Abstract:
+// - The Jumplist is the menu that pops up when right clicking a pinned
+// item in the taskbar. This class handles updating the Terminal's jumplist
+// using the Terminal's settings.
+//
+
+#pragma once
+
+#include "CascadiaSettings.h"
+#include "Profile.h"
+
+#include <ShObjIdl.h>
+
+class Jumplist
+{
+public:
+    Jumplist() = default;
+    void UpdateJumplist(std::shared_ptr<::TerminalApp::CascadiaSettings> settings);
+
+private:

```

Comment:
Jumplist doesn't seem to maintain any member variables. Should these functions just be static?